src/components/forms/common/AutocompleteDifficulties.tsx

```diff
@@ -2,16 +2,16 @@ import { Box } from '@mui/material';
 import LensIcon from '@mui/icons-material/Lens';
 import { AutocompleteElement } from 'react-hook-form-mui';
 
-import type { Difficulty } from '../../utils/Difficulty';
-import { Difficulties } from '../../utils/Difficulty';
+import type { Difficulty } from '../../../utils/Difficulty';
+import { Difficulties } from '../../../utils/Difficulty';
 
 const getOptionLabel = (option: Difficulty) =>
 	`${option.name} (${option.value}/${Difficulties.length})`;
 
 export const AutocompleteDifficulties = () => (
 	<AutocompleteElement
-		name="difficulties"
-		required
+		name="difficulty"
```

Comment:
Name conventions must correspond to parameters to be automatically mapped at the end